api.yml

```diff
@@ -0,0 +1,203 @@
+swagger: "2.0"
+
+info:
+  version: "0.1"
+  title: Kubernetes Queen API
+  description: A simple API to interact with Kubernetes clusters
+
+schemes:
+  - https
+  - http
+
+basePath: /api/v1
+
+definitions:
+  Cluster:
+    type: object
+    properties:
+      id:
+        type: "integer"
```

Comment:
usually it's better not to use int as id. some uuid (string) or human-readable (string as well) will be more flexible.

Reply:
uuid is a good idea, thanks!